tests/test_check_deleted_symbols.py

```
@@ -655,6 +755,64 @@ def test_reexport_dropped_from_init_fires(self, tmp_path: Path):
+        violations, waived, _ = cds.check_deleted_symbols(base_tip, repo)
+
+        assert violations == []
+        assert waived == set()
+
+    def test_py_to_symlink_losing_symbol_fires(self, tmp_path: Path):
+        """foo.py -> symlink to bar.py (which no longer defines func_a): func_a
+        is genuinely deleted and the gate names it."""
+        repo = tmp_path / "repo"
+        base_tip = self._build(repo, "bar.py", bar_has_func=False)
+
+        violations, waived, _ = cds.check_deleted_symbols(base_tip, repo)
```

Ruff (0.16.2)

[warning] Unpacked variable `waived` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)